intro-HPC/ch_best_practices.tex

```diff
@@ -37,9 +37,14 @@ \section{General Best Practices}
     have your default environment, so don't forget to load the software with
     \verb|module load|.
 
-  \ifgent\else\item  In case your job not running, use ``checkjob''.  It will show why your
+  \ifgent
+  \else
+  \ifbrussel
```

Comment:
@smoors missing `else` here, unless `checkjob` works at VUB?

Reply:
@boegel should be fixed now.